user-guide/Basic_Functionality/Visio/tutorials/Getting_started_with_basic_shapes.md

```diff
@@ -0,0 +1,136 @@
+
+1. Right-click the newly created ellipse and select *Data* > *Define Shape Data*.
+
+1. Set the *Label* field to `Element`.
+
+1. Set the *Value* field to the name of your element.
+
+1. Click *OK*.
+
+1. Save your Visio file.
+
+In DataMiner Cube, your Visual Overview should now be updated with your changes. You should see an interactive ellipse shape that is filled with the current alarm state color.
+
+![Basic element shape](~/user-guide/images/BasicElementShape.webp)
+
+## Step 3: Show more information efficiently
+
+1. In the *Tools* section of the *Home* tab of the ribbon, select the rectangle.
+
+1. Draw a rectangle on your page by clicking and dragging the mouse on it while this tool is selected.
+
+1. Right-click the newly created rectangle and select *Data* > *Define Shape Data*.
+
+1. Set the *Label* field to `Parameter`.
+
+1. Set the *Value* field to `350`.
+
+   This is the parameter ID of the *Total Processor Load* parameter.
+
+1. Click *OK*.
+
+1. Double-click the rectangle shape and enter the shape text `*`.
+
+1. Right-click the ellipse and select *Shape Data* to open the *Shape Data* panel.
+
+1. In the *Shape Data* panel, set the value of the "Element" shape data to "*".
+
+1. In the *Tools* section of the *Home* tab of the ribbon, select *Pointer Tool*.
+
+1. Select both shapes on your page, right-click and select *Group* > *Group*.
+
+   ![Group shapes](~/user-guide/images/GroupShapes.gif)
+
+1. Right-click the newly created shape group and select *Data* > *Define Shape Data*.
+
+1. Add shape data with label `Element` and set its value to the name of your element.
+
+1. Save your Visio file.
+
+In DataMiner Cube, you should see two colored shapes, one of which displays the value of your parameter.
+
+![Group shape result](~/user-guide/images/GroupShapeResult.webp)
+
+## Step 4: Clean up the drawing
+
+1. Right-click the shape group and select *Data* > *Define Shape Data*.
+
+1. Click the *New* button to add shape data with label `Enabled` and value `false`.
+
+1. Right-click the shape group and select *Group* > *Open Group*.
+
+1. Right-click the ellipse shape and select *Data* > *Define Shape Data*.
+
+1. Click the *New* button to add shape data with label `Enabled` and value `true`.
+
+1. Right-click the rectangle shape and select *Data* > *Define Shape Data*.
+
+1. Click the *New* button to add shape data with label `Options` and value `NoAlarmColorFill`.
+
+1. Make sure you have no shapes selected, click the "..." in the top-right corner of the Advanced Editing panel and select *Add pretty hover*.
+
+   ![Add pretty hover](~/user-guide/images/AddPrettyHover.gif)
+
+1. Save your Visio file.
+
+In DataMiner Cube, only your ellipse will now be colored with the alarm state of the element, and only that shape will be clickable. The pretty hover will ensure that the highlighted area upon hover of the ellipse follows the ellipse rather than its bounding rectangle.
```

Comment:
"The pretty hover will ensure that the highlighted area upon hover of the ellipse follows the ellipse rather than its bounding rectangle."
=> This sentence isn't very clear. Could you rephrase this?